OpenTextBC.js

```diff
@@ -0,0 +1,89 @@
+{
+	"translatorID": "8bdd3e33-87a0-488e-aa06-05bf87bea942",
+	"label": "openTextBC",
+	"creator": "Marielle Volz",
+	"target": "^https?://opentextbc\\.ca/[a-z0-9\\-]+/chapter/[a-z0-9\\-]+",
```

Comment:
...And we have a couple choices for how to approach matching URLs. We could do it like this and have a lot of false positives that we filter out with `detectWeb`:

```suggestion
	"target": "^https?://[^/]+/[a-z0-9\\-]+/chapter/[a-z0-9\\-]+",
```

Or we could try to come up with a regex that matches everything. A lot of Pressbooks sites are subdomains of `pressbooks.pub`, but not all; see a full list by expanding "Network" [here](https://pressbooks.directory/).

Reply:
Yeah, I'm not sure url matching will work for this - maybe I should just move this into the embedded metadata translator entirely instead of doing a separate translator, and check for <meta name="application-name" content="Pressbooks">?

Reply:
We don't want to hardcode workarounds for specific sites/apps into EM - that's what other translators are for. Using the regex I suggested with a more specific `detectWeb` should work fine.